testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -560,9 +561,39 @@ public void testMaterializedView()
                         "FROM\n" +
                         "  nation");
 
+        assertUpdate("DROP MATERIALIZED VIEW " + viewWithComment);
```

Comment:
why?
If it's needed. Add a comment why you do it

Reply:
Just because we don't need to use `viewWithComment` in the filtering tests since it is in the same schema as `view`. We only want to test filtering on the two MVs in different schemas.